src/core/app/index.ts

```diff
@@ -0,0 +1,16 @@
+import type { AtomBinded } from '@reatom/core';
+export type { AppConfig } from './types';
+export const appRegistry = {
+  atoms: {},
+  listeners: {},
+};
```

Comment:
```suggestion
export const appAtomsRegistry = {
  atoms: {},
  listeners: {},
};
```
?

Reply:
No, it's not planned as exclusively Atoms registry